src/data-store.ts

```diff
@@ -30,6 +32,9 @@ export const getCovidData = (placeName: string) => {
     let newPlaceName = countryNameApiMapping.has(placeName)
         ? countryNameApiMapping.get(placeName)
         : placeName;
+    if(!timeLineData.has(newPlaceName)) {
+        countryTimeLineCB(newPlaceName);
+    }
```

Comment:
Do we need to check if this is a country before we call the callback ?

Reply:
We will not no if it is a country or state. As the data for all states and US has been prepopulated we don't need to make the call. Also we don't need to make the call once the data has been populated